index.js

```diff
@@ -376,7 +376,7 @@ class BitbucketScm extends Scm {
             });
     }
 
-   /**
+    /**
```

Comment:
Why are the commenting patterns inconsistent? I'm seeing

```
/*
 *
 *
```

and

```
/*
*
*
```

Not really important, just wondering whether these whitespace changes were done by the linter, or done manually.